DataFormats/CTPPSDigi/src/CTPPSPixelDataError.cc

```diff
@@ -0,0 +1,98 @@
+
+#include "DataFormats/CTPPSDigi/interface/CTPPSPixelDataError.h"
+
+
+CTPPSPixelDataError::CTPPSPixelDataError() {}
+
+CTPPSPixelDataError::CTPPSPixelDataError(uint32_t errorWord32, const int errorType, int fedId) : 
+  errorWord32_(errorWord32),
+  errorType_(errorType),
+  fedId_(fedId)
+{
+  setMessage();
+}
+
+CTPPSPixelDataError::CTPPSPixelDataError(uint64_t errorWord64, const int errorType, int fedId) : 
+  errorWord64_(errorWord64),
+  errorType_(errorType),
+  fedId_(fedId)
+{
+  setMessage();
+}
+
+CTPPSPixelDataError::~CTPPSPixelDataError() {}
```

Comment:
@fabferro , current recommendation for empty destructors:
CTPPSPixelDataError::~CTPPSPixelDataError() = default

Reply:
done